src/utils/lines.js

```diff
@@ -1,7 +1,7 @@
-/* flow */
+/* @flow */
```

Comment:
> /* flow */ is incorrect flow comment. /* @flow */ is correct.

Yeah, i was asking, why you did it here, since this is unrelated change, and i guess i answered it too :)